pdns/recursordist/test-recursorcache_cc.cc

```diff
@@ -387,6 +388,7 @@ BOOST_AUTO_TEST_CASE(test_RecursorCacheSimpleDistantFuture)
 
 BOOST_AUTO_TEST_CASE(test_RecursorCacheGhost)
```

Comment:
Note that you could use a fixture [1] to do that, although in this particular case that might not be worth the trouble.

[1]: https://www.boost.org/doc/libs/1_83_0/libs/test/doc/html/boost_test/tests_organization/fixtures/case.html